content/quiz_process.md

```diff
@@ -7,19 +7,63 @@
         "2-Team 20-Question", "2-Team Overtime", and "3-Team Overtime".
 
 ### 3-Team Quiz
+- There will be 20 numbered questions in a quiz.
```

Comment:
I left this out "Each question will be read only once."    Do we want that?

Reply:
Lol no.

Reply:
I mean, if we remove it, it technically allows duplicate questions being asked.

Reply:
Districts often have further overrides (e.g. can't ask two questions from the same verse in a same quiz) so it's largely irrelevant. But removing it IS a functional change.

Reply:
I think it needs to stay. We need to purge from our minds the "everybody knows this" or "nobody does this" concepts. Think of the rulebook as software code.